src/server.c

```diff
@@ -6475,7 +6475,8 @@ void loadDataFromDisk(void) {
         int ret = loadAppendOnlyFiles(server.aof_manifest);
         if (ret == AOF_FAILED || ret == AOF_OPEN_ERR)
             exit(1);
-        serverLog(LL_NOTICE, "DB loaded from append only file: %.3f seconds", (float)(ustime()-start)/1000000);
+        if (ret == AOF_OK || ret == AOF_TRUNCATED)
```

Comment:
on AOF_NOT_EXIST we should indeed skip the print, but maybe on AOF_EMPTY we should still print it?
i.e. an empty AOF file is logically a valid one...
on the other hand, now that we always create an empty "preamble" RDB base file. the AOF file will never be empty (other than upgrades from previous releases), right?

Reply:
Yes, you are right.